src/index.js

```diff
@@ -374,19 +374,18 @@ const fullWalletFromTicket = async config => {
     seed: buf2hex(ownerSeed),
   }
 
-  let manageSeeds = {};
   const manageNodes = await Promise.all(ships.map(async ship => {
-    let res = await childNodeFromSeed({
-      seed: ownerSeed,
-      type: 'manage',
-      revision: _revisions.manage,
-      ship: ship,
-      password: password,
+     return await childNodeFromSeed({
```

Comment:
Pretty sure we can then do away with the `async` and `{ return await` and `}` to get us back to that lean, mean, functional aesthetic.

Great change @jtobin!

Reply:
@Fang- right you are.  Slash them LoC!

Reply:
For the record, I am generally in favor of putting in the optional `{}` and `return`, for the same reason I'm in favor of optional `()`. But right here and now, that would be inconsistent with the rest of the codebase.

Reply:
I may opine on this topic in a future issue of the Digest.  In general, I think matching the surrounding style of the code is the most important thing.  People can argue, very reasonably on all sides, that something should or should not be this way or that, syntactically or stylewise or what have you.

(Except tabs vs spaces, of course -- people who advocate tabs are beyond the reach of reason.)

But I think the author/maintainer of a codebase ought set the style, whatever that is, and that contributors ought match it.  An author/maintainer who enforces no particular style, or who accepts contributions of any style willy-nilly, is probably doing everyone a disservice.